oak/server/rust/oak_runtime/src/node/wasm.rs

```diff
@@ -877,35 +877,92 @@ impl super::Node for WasmNode {
     }
 }
 
-#[test]
-fn wasm_invalid_args() {
-    use super::Node;
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::node::Node;
     use crate::runtime::TEST_NODE_ID;
```

Comment:
Is it necessary to have TEST_NODE_ID declared in runtime? Or can I move it to this module?

Reply:
`TEST_NODE_ID` is used by `sdk/rust/oak_tests` so I think the declaration in runtime is the best place. Future tests in other parts of `oak_runtime` may also need to use `TEST_NODE_ID`